src/Modules/Manage/Dnn.PersonaBar.Users/Components/Prompt/Commands/DeleteUser.cs

```diff
@@ -56,21 +58,18 @@ public override ConsoleResultModel Run()
                 return new ConsoleErrorResultModel(LocalizeString("Prompt_UserAlreadyDeleted"));
             }
 
-            if (!UserController.DeleteUser(ref userInfo, Notify, false))
+            if (!_userControllerWrapper.DeleteUserAndClearCache(ref userInfo, Notify, false))
```

Comment:
How about to save portalId in local variable before passing as ref type parameter which could portalId?

Reply:
It would not change anything.

Reply:
Oh wait, you mean to keep portalId variable. You could do that if you think that the value on the userInfo would change